assemblies/data-access-plugin/src/main/resources/settings.xml

```diff
@@ -19,6 +19,10 @@
   <!-- default view acls for user or role -->
   <data-access-default-view-acls>31</data-access-default-view-acls>
 
+  <!-- Uncomment and set the number of threads to load the datasources in Manage Data sources Default number of threads
```

Comment:
the jira issue id in the commit comment needs to be enclosed in square brackets.
For example:[BACKLOG-35854]

The release team uses a script to grab all the jira issues in release, and they use the regex with [] to find out this information.